extensions/wikia/Recirculation/services/DiscussionsDataService.class.php

```diff
@@ -99,6 +99,7 @@ private function buildUrl( $endpoint, $options ) {
 			'limit' => self::DISCUSSIONS_API_LIMIT,
 			'sortKey' => self::DISCUSSIONS_API_SORT_KEY,
 			'sortDirection' => self::DISCUSSIONS_API_SORT_DIRECTION,
+			'viewableOnly' => true,
```

Comment:
Minor: looks like the pattern here is to define the value as a constant of this class. Could we do that here as well to keep things consistent?